### PR TITLE
rouge-score 0.1.2 :snowflake:

### PR DESCRIPTION
**Pull request by ifitchet**
rouge-score 0.1.2 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-4823]
- dev_url:        https://github.com/google-research/google-research/tree/master/rouge
- conda_forge:    https://github.com/conda-forge/rouge-score-feedstock/blob/main/recipe/meta.yaml
- pypi:           https://pypi.org/project/rouge-score/0.1.2
- pypi inspector: https://inspector.pypi.io/project/rouge-score/0.1.2

### Explanation of changes:

- clone of conda-forge recipe
- > [!WARNING] no upstream tests, the upstream is 1.3GB of monorepo with no tags or tarballs thus no sane way to get the testdata to see if it works
- copy individual testdata file from GH
- skip tests on Windows (re-use of temporary files)

[PKG-4823]: https://anaconda.atlassian.net/browse/PKG-4823?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -67,6 +112,10 @@ about:
 
     PyPI: [https://pypi.org/project/rouge-score/](https://pypi.org/project/rouge-score/)
 
+  # There's no clear doc_url other than this README
+  doc_url: https://github.com/google-research/google-research/tree/master/rouge
```
Comment (ViridianMelody) [inline recipe/meta.yaml]:
```suggestion
  doc_url: https://github.com/google-research/google-research/blob/master/rouge/README.md
```

**Review by ViridianMelody — APPROVED** [review]
Well done getting some of the tests running! Minor suggestion to link directly to the readme for doc_url

**Review by skupr-anaconda — APPROVED** [review]
LGTM but Melody's comment

File: recipe/meta.yaml (hunk 1 above)
Comment (ifitchet) [reply recipe/meta.yaml]: Great spot!

**Pull request merged**